pkg/client/jiva/controller_client.go

```diff
@@ -146,3 +146,30 @@ func (c *ControllerClient) GetVolumeStats(address string, obj interface{}) (int,
 	rc := json.NewDecoder(resp.Body).Decode(obj)
 	return 0, rc
 }
+
+// GetVolumeAccessMode only gets the access mode of volume, i.e, whether it is RW,
+// RO etc which is coming from the jiva controller.This is the limitation because
+// of the response we are recieving from the endpoint /v1/replicas.
+func GetVolumeAccessMode(address string, obj interface{}) error {
```

Comment:
There is already a method `GetVolumeStats( )` available to make a API request to jiva controller.
Can we write a new method which will construct the API url and use `GetVolumeStats( )` method, so we can re-use our code.?